blacken.el

```diff
@@ -145,13 +145,45 @@ Show black output, if black exit abnormally and DISPLAY is t."
              (when display
                (pop-to-buffer errbuf))))))
 
+(defun find-nearest-file (filename &optional dir)
```

Comment:
You can simple use `locate-dominating-file`

Reply:
Thanks! This was very helpful.